puro/lib/src/version.dart

```diff
@@ -80,8 +81,11 @@ class PuroVersion {
       final packages = packageData['packages'] as List<dynamic>;
       final puroPackage = packages
           .cast<Map<String, dynamic>>()
-          .firstWhere((e) => e['name'] == 'puro');
-      final rootUri = Uri.parse(puroPackage['rootUri'] as String);
+          .firstWhereOrNull((e) => e['name'] == 'puro');
+      if (puroPackage == null) {
+        return null;
+      }
+      final rootUri = Uri.parse(puroPackage?['rootUri'] as String);
```

Comment:
One final thing and it should be ready, the `?` in `final rootUri = Uri.parse(puroPackage?['rootUri'] as String);` on line 88 generates a warning because puroPackage can't be null anymore, it can be removed

Reply:
Done.